test_schemas/en/test_custom_page_titles.json

```diff
@@ -0,0 +1,841 @@
+{
```

Comment:
We should also test for a question block with a custom page title and no placeholders.

Reply:
Done in ef1ed219

Reply:
I can't see a question block in the schema with a page title and no placeholders.